recipes/persim/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "persim" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 07b14bd7e17b187c760297954b55788396b6be31e1e9ead35e3d73ebfa488771
+  
+build:
+  noarch: python
+  number: 0
+  script: package_version={{ version }} python -m pip install --no-deps --ignore-installed .           # [not win]
+  script: set "package_version={{ version }}" && python -m pip install --no-deps --ignore-installed .  # [win]
```

Comment:
Please change it to:
```
script:
  - package_version={{ version }} python -m pip install --no-deps --ignore-installed .           # [unix]
  - set "package_version={{ version }}" && python -m pip install --no-deps --ignore-installed .  # [win]
```

Reply:
I was wondering, why do you need ``package_version`` since you already have defined the version in your ``_version.py`` ?

Reply:
Good question. Only because I am piecing together build scripts from other projects.  I'm having trouble finding resources for learning how to do this correctly. Any pointers would be great 💛